<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSearchControl</class>
 <widget class="QWidget" name="FormSearchControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>315</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="4">
      <widget class="QPushButton" name="buttonMore">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QLineEdit" name="lineStart">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Threads:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4" colspan="3">
      <widget class="QCheckBox" name="checkStop">
       <property name="toolTip">
        <string>stop as soon as the next set of matching seeds is found</string>
       </property>
       <property name="text">
        <string>Stop on results</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;searching algorithm&lt;/p&gt;&lt;p&gt;incremental: best for non-exhaustive biome searches&lt;/p&gt;&lt;p&gt;48-bit family blocks: best for searches with strong structure restrictions&lt;/p&gt;&lt;p&gt;seed list from file...: load and search through a pre-computed list of seed&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Search:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="buttonSearchHelp">
       <property name="maximumSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../rc/icons.qrc">
         <normaloff>:/icons/info.png</normaloff>:/icons/info.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Seed:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QSpinBox" name="spinThreads">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QComboBox" name="comboSearchType">
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>16</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>incremental</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>48-bit family blocks</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>seed list from a file...</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0" colspan="7">
      <widget class="QProgressBar" name="progressBar">
       <property name="toolTip">
        <string>Queued progress within search set</string>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="format">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="7">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="buttonClear">
         <property name="text">
          <string>Clear results</string>
         </property>
         <property name="icon">
          <iconset resource="../rc/icons.qrc">
           <normaloff>:/icons/clear.png</normaloff>:/icons/clear.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonStart">
         <property name="text">
          <string>Start search</string>
         </property>
         <property name="icon">
          <iconset resource="../rc/icons.qrc">
           <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0" colspan="7">
      <widget class="QLabel" name="labelStatus">
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Idle</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTableView" name="results">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
